src/main/scala/com.socrata.tileserver/services/TileService.scala

```diff
@@ -169,10 +168,10 @@ object TileService {
                     select: String): Map[String, String] = {
     val params = req.queryParameters
     val whereParam =
-      if (params.contains("$where")) s"""(${params("$where")}) and (${where})""" else where
+      if (params.contains(s"$$where")) s"""(${params(s"$$where")}) and (${where})""" else where
```

Comment:
This is because we have `-Xfatal-warnings` enabled, and 2.11 now warns on `$` that aren't part of an interpolated string.

(And scalac doesn't appear to have an easy way to suppress warnings.)